response_operations_ui/views/collection_exercise.py

```diff
@@ -405,6 +413,79 @@ def create_collection_exercise(survey_ref, short_name):
                                 new_period=form.get('period')))
 
 
+@collection_exercise_bp.route('/<short_name>/<period>/<ce_id>/create-event/<tag>', methods=['GET'])
+@login_required
+def get_create_collection_event_form(short_name, period, ce_id, tag):
+    logger.info("Retrieving form for create collection exercise event", short_name=short_name, period=period,
+                ce_id=ce_id, tag=tag)
+
+    survey = survey_controllers.get_survey(short_name)
+
+    form = EventDateForm()
+    event_name = get_event_name(tag)
+
+    logger.info("Successfully retrieved form for create collection exercise event",
+                short_name=short_name,
+                period=period,
+                ce_id=ce_id, tag=tag)
+
+    return render_template('create-ce-event.html',
+                           short_name=short_name,
+                           period=period,
+                           survey=survey,
+                           event_name=event_name,
+                           form=form)
+
+
+@collection_exercise_bp.route('/<short_name>/<period>/<ce_id>/create-event/<tag>', methods=['POST'])
+@login_required
+def create_collection_exercise_event(short_name, period, ce_id, tag):
+    logger.info("Creating collection exercise event",
+                short_name=short_name,
+                period=period,
+                collection_exercise_id=ce_id,
+                tag=tag)
+
+    form = EventDateForm(request.form)
+
+    if not form.validate():
+        return get_create_collection_event_form(short_name, period, ce_id, tag, errors=form.errors)
+
+    day = form.day.data if not len(form.day.data) == 1 else f"0{form.day.data}"
```

Comment:
Remove this line, we can specify the format of the date in the string interpolation